package.json

```diff
@@ -69,8 +69,8 @@
     "lodash": "^4.17.4",
     "mysql": "^2.13.0",
     "mz": "^2.6.0",
-    "pg-promise": "^6.3.6",
-    "typescript": "^2.7.1",
+    "pg-promise": "^10.11.0",
+    "typescript": "^4.4.3",
```

Comment:
projectaussie has:

     "typescript": "4.2.3",
     "pg-promise": "10.10.2",
     
We should likely not require higher than those.

Reply:
Done!